views/download.erb

```diff
@@ -24,11 +24,11 @@
     <div class="col-md-9">
       <div class="headline"><h2>Fluentd (v1, current stable)</h2></div>
       <h3>RPM, deb, Windows, macOS</h3>
-      <p>Fluentd v1 is available on Linux, Mac OSX and Windows. It's packaged by <a href="https://www.treasuredata.com/">Treasure Data</a> and <a href="https://calyptia.com/">Calyptia</a> respectively as:</p>
+      <p>Fluentd v1 is available on Linux, Mac OSX and Windows. It's packaged by <a href="https://www.fluentd.org/">Fluentd Project</a> and <a href="https://calyptia.com/">Calyptia</a> respectively as:</p>
       <ul class="list-unstyled">
         <li>
 	  <i class="icon-arrow-right color-green"></i>
-	  <a href="/download/td_agent">Treasure Agent (td-agent)</a>
+	  <a href="/download/fluent_package">Fluent Package (fluent-package)</a>
```

Comment:
We should add also the word `td-agent` to guide existing users:
```suggestion
	  <a href="/download/fluent_package">Fluent Package (fluent-package), formerly known as Treasure Agent (td-agent)</a>
```

Reply:
Thanks!